### ISSUE TITLE
Incorrect main file path in bower.json 

### ISSUE THREAD

**Issue by pinicarus** [craftyjs/Crafty#732]
The bower.json file for release 0.6.2 has the main file set to "crafty.js", but the file is actually located in "dist/crafty.js". This makes it difficult to interact with e.g. grunt tasks.

Can you please consider fixing this and republishing ?
Thanks.

**Comment by starwed** [comment]
I think I'd rather simply point bower to a separate repo that exists just for that purpose.

Publication data shouldn't be so inextricably tied up with an actual release, and this is like the third or fourth time some issue like this has come up.  Since I don't personally use bower at all, it's hard for me to know when it's actually working or not, and it seems impossible to test in the actual development branch because of its weird requirements.  AFAICT, to know if it's working, you have to do an actual release, which seems like madness...

**Comment by ahuth** [comment]
I know this is an older issue, but it's still open, so here's my thoughts:

Bower distribution stuff and active development shouldn't be in the same repo. I'm not sure what the best way to split those up is, though.

What you can do is specify an override in your own bower.json:

``` js
{
  ...
  "dependencies": {
    "crafty": "~0.6.2"
  },
  "overrides": {
    "crafty": {
      "main": "dist/crafty.js"
    }
  }
  ...
}
```

Now Grunt or Gulp will be able to figure out where to get the file from.

**Comment by starwed** [comment]
> Bower distribution stuff and active development shouldn't be in the same repo.

I've seen some other projects that simply have a repo that contains just the files bower needs.  For us that would, I guess be only the regular and minified distro files.  Perhaps also some compiled documentation?  Without having used bower, I'm kind of flying blind here.  :)

**Comment by navelpluisje** [comment]
Hi starwed,
I'm using Bower and had the same issue as the one mentioned above. It is also better to seperate some stuff for bower. But you do not need another repo for that. Within the Bower.json you can add a part named `ignore`. All the files and directories mentioned there will not be fetched by bower. Here's an example of how the file should look like:

```
{
  "name": "crafty",
  "version": "0.6.2",
  "main": "dist/crafty.js",
  "license": "MIT",
  "ignore": [
    "**/.*",
    "build",
    "playgrounds",
    "test",
    "*.md",
    "changelog.txt",
    "Gruntfile.js",
    "package.json"
  ]
}
```

This way only the `dist` and `src` directories will be fetched by bower with the bower file.

I hope you will modify this one the next release.

**Comment by mucaho** [comment]
fixed in #825 .
